tests/lib.rs

```diff
@@ -607,3 +607,63 @@ fn deploy_fails_with_floats() {
         Err(rusk_vm::VMError::InstrumentalizationError(_))
     ));
 }
+
+#[test]
+fn persistance() {
+    use microkelvin::DiskBackend;
+
+    let counter = Counter::new(99);
+
+    let code =
+        include_bytes!("../target/wasm32-unknown-unknown/release/counter.wasm");
+
+    let contract = Contract::new(counter, code.to_vec());
+
+    let (persist_id, contract_id) = {
+        let mut network = NetworkState::default();
+
+        let contract_id = network.deploy(contract).unwrap();
+
+        let mut gas = GasMeter::with_limit(1_000_000_000);
+
+        assert_eq!(
+            network
+                .query::<_, i32>(contract_id, counter::READ_VALUE, &mut gas)
+                .unwrap(),
+            99
+        );
+
+        network
+            .transact::<_, ()>(contract_id, counter::INCREMENT, &mut gas)
+            .unwrap();
+
+        assert_eq!(
+            network
+                .query::<_, i32>(contract_id, counter::READ_VALUE, &mut gas)
+                .unwrap(),
+            100
+        );
+
+        (
+            network
+                .persist(|| DiskBackend::new("./"))
```

Comment:
1. Never use explicit path such "./", use `home_dir` instead. But for tests, you should always use the `tempdir`.

2. Did you ensure that the file is removed at the end of the tests? AFAIK only `ephemeral` automatically delete the file during the drop, and we cannot use it (we have to delete the file at the end of the test).